sharktuner/model_tuner/sharktuner.py

```diff
@@ -6,11 +6,11 @@
 
 import argparse
 from pathlib import Path
-from tuner import libtuner
-from tuner.common import *
+from sharktuner import libtuner
+from sharktuner.common import *
 
 
-class SimpleTuner(libtuner.TuningClient):
+class SharkTuner(libtuner.TuningClient):
```

Comment:
Actually, shouldn't this be called `ModelTuner`? The issue I see is that now we have two sharktuners -- on top level, the other one here

Reply:
Beyond that, is this still considered as an example and should it go into the package?

Reply:
It should go into the package

Reply:
Right now it is not. Actually, building the wheel fails with the current branch.